lib/resources/aws/aws_vpc.rb

```diff
@@ -30,8 +30,8 @@ def validate_params(raw_params)
       allowed_scalar_type: String,
     )
 
-    if validated_params.key?(:vpc_id) && validated_params[:vpc_id] !~ /^vpc\-[0-9a-f]{8}/
-      raise ArgumentError, 'aws_vpc VPC ID must be in the format "vpc-" followed by 8 hexadecimal characters.'
+    if validated_params.key?(:vpc_id) && validated_params[:vpc_id] !~ /^rtb\-([0-9a-f]{8})|(^rtb\-[0-9a-f]{17})$/
```

Comment:
small typo :joy: , should have been `vpc` instead of `rtb`. I see you copied from the old PR :joy:

Reply:
oops.! ))